lampod/src/ln/peer_manager.rs

```diff
@@ -203,6 +204,7 @@ impl LampoPeerManager {
             }
             result
         });
+        result.join().unwrap()?;
```

Comment:
Why this change? you are blocking lampo so from this join the code will never return, Have you tested the code?

Reply:
I still need to see this, Will do in this week!

Reply:
Take your time, this will be in the august release